tests/unit/data_classes/test_api_gateway_authorizer_event.py

```diff
@@ -52,16 +52,16 @@ def test_api_gateway_authorizer_v2():
     assert parsed_event.path_parameters == raw_event["pathParameters"]
     assert parsed_event.stage_variables == raw_event["stageVariables"]
 
-    assert parsed_event.get_header_value("Authorization") == "value"
-    assert parsed_event.get_header_value("authorization") == "value"
-    assert parsed_event.get_header_value("missing") is None
+    assert parsed_event.headers["Authorization"] == "value"
```

Comment:
I was thinking about using `.get()`, but this is a test I think doesnt matter here. What do you think?